src/xcode/ENA/HealthCertificateToolkit/Tests/HealthCertificateToolkitTests/Validation/OnboardedCountriesTests.swift

```diff
@@ -0,0 +1,24 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import SwiftCBOR
+import XCTest
+@testable import HealthCertificateToolkit
+
+class OnboardedCountriesTests: XCTestCase {
+
+    func test_CreateOnboardedCountries() throws {
```

Comment:
unhappy path?
```
CBOR.utf8String("")
CBOR.utf8String("XX")
CBOR.utf8String("🏴‍☠️")
```

Reply:
Yeah a test with wrong values would be nice

Reply:
Okay it would make sense to handle the error in the service who is using this functionality. So the return values can be everything and thus we do not need to handle any errors here.